### PR TITLE
refactor: unify the error handling methods in the crypto package that are different from the project style

### PR DESCRIPTION
**Pull request by ChengenH**

### CODE REVIEW

**Review by gballet — CHANGES_REQUESTED** [review]
There's no point in checking for `err == nil` if you're going to use `!errors.Is`

**Comment by ChengenH** [comment]
> > There's no point in checking for `err == nil` if you're going to use `!errors.Is`
> 
> Thanks, I'll fix it

**Comment by ChengenH** [comment]
> There's no point in checking for `err == nil` if you're going to use `!errors.Is`
I have already processed it, Please review again, thank u very much

File: cmd/clef/main.go [hunk 1]
```diff
@@ -923,13 +923,13 @@ func testExternalUI(api *core.SignerAPI) {
 		}
 	}
 	expectApprove := func(testcase string, err error) {
-		if err == nil || err == accounts.ErrUnknownAccount {
+		if errors.Is(err, accounts.ErrUnknownAccount) {
```
Comment (rjl493456442) [inline cmd/clef/main.go:926]: The behavior is changed here. If err == nil, `errors.Is(err, accounts.ErrUnknownAccount)` will be false.

Comment (ChengenH) [reply cmd/clef/main.go]: > The behavior is changed here. If err == nil, `errors.Is(err, accounts.ErrUnknownAccount)` will be false.

I'm fixed

**Comment by ChengenH** [comment]
> I don’t think this change is worthwhile. It’s essentially unnecessary, and our principle is to avoid code style changes unless they are absolutely useful.

Ok, if this change is not worth it, I will close it. Thank you!

**Pull request closed without merging**